### ISSUE TITLE
FBX Export To Unity Does Not Export Textures Despite Following Manual Instructions

### ISSUE THREAD

**Issue by simonpaulkeating** [JannisX11/blockbench#2296]
### What are you trying to do, and what do you expect to happen?

When exporting a model to Unity using the fbx (or .obj) export, the associated textures do not get exported.

Although I followed the instructions as set out in the manual and extracted the textures and materials from the imported prefab inside Unity, the texture file was empty and so the material simply shows a default grey material. I tried this with multiple textured models. 

I am using the latest Unity version LTS 2022.3.22f1

### What happens instead?

When imported the texture file was empty and so the material simply shows a default grey material. 

The only way to get it working is to drag in the texture manually and assign then assign it to the material. This method is time consuming.

### Model format in which the issue occurs

Generic Model

### Blockbench variant

Program

### Blockbench version

4.9.4

### Operating System

Windows 10

### Installed Blockbench plugins

None

**Comment by JannisX11** [comment]
Interestingly Sketchfab does load the textures correctly.

**Comment by simonpaulkeating** [comment]
Yeah I noticed it worked in Windows paint3d too, so the data is there somewhere I guess.
Things I tried that didn't work include... Cinema4d, Blender and Unity.

**Comment by Nineforcestudios** [comment]
With Unity Cause i also use unity Create a new material and drag the texture into the material then apply to your prefab and it should work
